app/useAnalytics.js

```diff
@@ -0,0 +1,276 @@
+/** Usage:
+ * 
+   * Firebase Analytics Documentation Example from Donate route:
+   * usePageAnalytics("/donate") from useAnalytics.js logs dead clicks and load times for the /donate page
+   
+  usePageAnalytics("/donate");
+
+  * logButtonEvent from analytics.js logs a button click event to Firebase Analytics when the 
+    "Make a Donation" button is clicked. The event is labeled as "make donation button clicked" 
+    and is associated with the "/donate" page.
+
+    <Button
+    btnText="Make a Donation"
+    color="bg-blue-600"
+    textColor="text-white"
+    hoverColor="hover:bg-blue-700"
+    rounded="rounded-md"
+    font="font-semibold"
+    onClick={() =>
+      logButtonEvent("make donation button clicked", "/donate")
+    }
+    />
+ */
+import { useEffect } from "react";
+import {
+  logPageView,
+  logError,
+  logDeadClick,
+  logInternetDisconnection,
+  logLoadingTime,
+} from "./utils/analytics";
+import { useReportWebVitals } from "next/web-vitals";
+import { ref, getDownloadURL, uploadString } from "@firebase/storage";
+import { storage } from "./firebaseConfig";
+
+async function uploadScreenshot(base64Image, fileName) {
+  const storageRef = ref(storage, `deadclicks/${fileName}.png`);
+  await uploadString(storageRef, base64Image, "data_url");
+  return await getDownloadURL(storageRef);
+}
+
+const captureClickArea = async function(normalizedX, normalizedY, radius = 100) {
```

Comment:
```suggestion
const captureClickArea = async function(normalizedX, normalizedY, radius = 300) {
```
I saw a few of them turned out to be too blank...Can we please extend the radius maybe by 3 fold that roughly increases the size by 10x (given that it's a square of 3)?